scripts/buildtools/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/algorand/go-algorand/scripts/buildtools
 
-go 1.16
+go 1.17
 
 require (
 	github.com/algorand/msgp v1.1.50
```

Comment:
This is where you'll update msgp after your other PR is merged and tagged

Reply:
@Eric-Warehime please update the version and I'll merge this

Reply:
Updated. Also had to run `go get` and `go mod tidy` which ended up separating deps into separate `require` blocks. See https://go-review.googlesource.com/c/go/+/325922/ and https://github.com/golang/go/issues/45965 for details.